cypress/integration/work-order/update-with-adding-multiple-sors.spec.js

```diff
@@ -0,0 +1,54 @@
+describe('Updating a work order', () => {
```

Comment:
We've got two update specs now so I'd say it's time to make a directory like we have for `create` for this and the existing update spec, under `cypress/integration/work-order/update/`